ImageData/CartaHdf5Image.cc

```diff
@@ -21,8 +21,8 @@ CartaHdf5Image::CartaHdf5Image(
     : casacore::ImageInterface<float>(casacore::RegionHandlerHDF5(GetHdf5File, this)),
       _valid(false),
       _pixel_mask(nullptr),
-      _mask_spec(mask_spec) {
-    _lattice = casacore::HDF5Lattice<float>(casacore::CountedPtr<casacore::HDF5File>(new casacore::HDF5File(filename)), array_name, hdu);
```

Comment:
I'm confused by this diff -- this change should already have been merged into both dev and the release.